builder

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env perl
 
 use Getopt::Long;
+use List::MoreUtils qw(uniq);
```

Comment:
Keep in mind that before we was using https://github.com/Sabayon/devkit/blob/master/builder#L107